.github/workflows/pr-comment.yaml

```diff
@@ -63,7 +63,7 @@ jobs:
         with:
           issue-number: ${{ steps.pr-number.outputs.pr-number }}
           comment-author: "github-actions[bot]"
-          body-includes: PR Check Results
+          body-includes: "## `ruff-ecosystem` results"
```

Comment:
Can we instead add a hidden marker in the comment and use it here? That way we don't have to update it in the future :)

Reply:
Well they both come from this job, I just didn't realize. A hidden marker sounds kind of nice.

Reply:
I've done that with a comment now

Reply:
Thanks!